ospo-mindmap/Content/ospomindmap.md

```diff
@@ -109,24 +189,87 @@ incubation and post launch
   - Vet, organize and operate OS projects
   - Train and coach OS Leaders
 
-### OS Leadership
+#### Optimize Open Outbound Source Contributions
+
+
+### 🤝 Collaborate with Open Source Organizations
+
+##### 📖 Open Standards
+
+- OpenChain
+- SPDX
+
+##### 🏠 Foundations
+
+- Apache Foundation
+- Free Software Foundation Europe
+  - Legal Network
+- Eclipse Foundation
+- Linux Foundation
+  - FINOS
+  - OpenSSF
+  - OpenChain
+  - SPDX
+  - TODO Group
+- Open Source Initiative
+- OW2
+  - Good Governance Initiative
 
-#### Provides open source consultancy and become a strategic partner
 
-- Evaluate the behavior
-and performance of the project
-- Advise on which open source technologies
-to adopt or remove
-- Provide insights into OSS trends
+### ⏱️ Track Performance Metrics
```

Comment:
@anajsana Under the section 'Develop and Execute Open Source Strategy' I already listed 'InnerSource' to internal metrics but on second thought maybe we should add metrics as it's own point. I like that @ibrahimhaddad in his [OSPOs PDF](https://github.com/ibrahimhaddad/publications/blob/master/Open%20Source%20Program%20Offices.pdf) made metrics a separate point instead of putting it under the other responsibilities because as he wrote "traditional metrics as used in product organizations, do not apply in the context of open source development". By having it as its own point it will hopefully trigger discussions in OSPO and highlight the importance of it  - the section need to be improved so it more clear why it stands on its own.

Reply:
+1 on having them as its own point. However, I still think that if we want to give them the importance they deserve, I think that adding them within a bigger picture that is a "metrics strategy" would be beneficial. Sometimes, organizations just start adding metrics, feel overwhelmed, and don't get to use them, because they never thought of a metrics strategy at the very beginning. So something like:

### ⏱️ Develop A Metrics Strategy
  - Track Performance Metrics